misc/python/materialize/xcompile.py

```diff
@@ -184,7 +180,7 @@ def _bootstrap_darwin_x86_64() -> None:
         contents = ""
     if contents == BOOTSTRAP_VERSION:
         return
-    spawn.runv(["brew", "install", "SergioBenitez/osxct/x86_64-unknown-linux-gnu"])
-    spawn.runv(["rustup", "target", "add", "x86_64-unknown-linux-gnu"])
+    spawn.runv(["brew", "install", f"messense/macos-cross-toolchains/{target(arch)}"])
```

Comment:
works for all cases! nice!

Reply:
oh wait, we dont need the rustup anymore?

Reply:
I know right!

Reply:
We do! It's still there?

Reply:
Oh I am bad at reading stuff on github apparently!

Reply:
No worries, glad I'm not crazy!